src/group_theory/schur_zassenhaus.lean

```diff
@@ -5,27 +5,27 @@ Authors: Thomas Browning
 -/
 
 import group_theory.complement
-import group_theory.index
+import group_theory.sylow
 
 /-!
 # Complements
 
-In this file we prove the Schur-Zassenhaus theorem for abelian normal subgroups.
+In this file we prove the Schur-Zassenhaus theorem.
 
 ## Main results
 
-- `exists_right_complement_of_coprime` : **Schur-Zassenhaus** for abelian normal subgroups:
-  If `H : subgroup G` is abelian, normal, and has order coprime to its index, then there exists
-  a subgroup `K` which is a (right) complement of `H`.
-- `exists_left_complement_of_coprime` : **Schur-Zassenhaus** for abelian normal subgroups:
-  If `H : subgroup G` is abelian, normal, and has order coprime to its index, then there exists
-  a subgroup `K` which is a (left) complement of `H`.
+- `exists_right_complement'_of_coprime` : If `H : subgroup G` is normal and has order coprime to
```

Comment:
we should still have the bolded name - it's really good for searching. also probably good to have it in the tags